robot_smach_states/test/human_interaction/answer_questions.py

```diff
@@ -0,0 +1,36 @@
+# System
+import unittest
+
+# TU/e Robotics
+from robot_skills.mockbot import Mockbot
+
+# Robot smach states
+from robot_smach_states.human_interaction.answer_questions import HearAndAnswerQuestions
+
+
+class TestHearAnswerQuestions(unittest.TestCase):
```

Comment:
This test currently fails

Reply:
It succeeds for me, what is the error you get?

Reply:
CI succeeds

Reply:
error message I get when running 'python answer_questions.py' is:

FAIL: test_teams_name (__main__.TestHearAnswerQuestions)

Traceback (most recent call last):
  File "answer_questions.py", line 32, in test_teams_name
    self.assertIn("tech united", self.robot.speech.speak.mock_calls[0].args[0].lower())
AssertionError: 'tech united' not found in ''

Ran 1 test in 0.044s

FAILED (failures=1)

Reply:
No issues for me.

Reply:
That's odd indeed. For me it worked when I wrote (and hence tested) it...

Reply:
I know Sam had the same issue, which I could not figure out. I thought it was compiler related, but I am not sure.